### ISSUE TITLE
ModuleNotFoundError: No module named 'click._bashcomplete'

### ISSUE THREAD

**Issue by michaelaye** [tiangolo/typer-cli#53]
Just installed typer and typer-cli for trying out.

Env: Python 3.8 conda env, installed typer and typer-cli via conda

```
╰─ mamba list click                                                                                                                 ─╯
# packages in environment at /home/maye/miniconda3/envs/py38:
#
# Name                    Version                   Build  Channel
click                     8.0.3            py38h578d9bd_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
(py38) 
╭─ ~/Dropbox/src/uvis_pdart_project  main ?1 ········································································ ✔  14:43:54 ─╮
╰─ mamba list typer                                                                                                                 ─╯
# packages in environment at /home/maye/miniconda3/envs/py38:
#
# Name                    Version                   Build  Channel
typer                     0.4.0              pyhd8ed1ab_0    conda-forge
typer-cli                 0.0.12             pyh6c4a22f_1    conda-forge
```

Trying to use `typer script.py run` I get this error, despite click being installed:

```
typer create_meeting_notes.py run                                                                                                ─╯
Traceback (most recent call last):
  File "/home/maye/miniconda3/envs/py38/bin/typer", line 6, in <module>
    from typer_cli.main import main
  File "/home/maye/miniconda3/envs/py38/lib/python3.8/site-packages/typer_cli/main.py", line 11, in <module>
    from click._bashcomplete import get_choices as original_get_choices  # type: ignore
ModuleNotFoundError: No module named 'click._bashcomplete'
```

**Comment by musicinmybrain** [comment]
This is the same as https://github.com/tiangolo/typer-cli/issues/53.

**Comment by michaelaye** [comment]
this **IS** #53 ?

**Comment by musicinmybrain** [comment]
> this **IS** #53 ?

Ha! That’s funny. I meant to say that this appears to be the same as https://github.com/tiangolo/typer-cli/issues/50, in which the current version of typer-cli still only works with typer 0.3.x and click 7.x.

**Comment by idvorkin** [comment]
@michaelaye  - The root cause here is typer-cli doesn't support click 0.8, can you rename the title like: Make typer-cli support click 8 (see required link above).

**Comment by tiangolo** [comment]
Thanks for the report and discussion everyone! :coffee: 

This should be solved by Typer-CLI `0.0.13`, just released :rocket: 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
